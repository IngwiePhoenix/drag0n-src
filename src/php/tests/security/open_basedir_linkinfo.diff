003+ 
004+ Warning: is_dir(): open_basedir restriction in effect. File(/Users/Ingwie/Work/drag0n-php/tests/security/test/ok/symlink.txt) is not within the allowed path(s): (.) in /Users/Ingwie/Work/drag0n-php/tests/security/open_basedir.inc on line 43
005+ 
006+ Warning: unlink(): open_basedir restriction in effect. File(/Users/Ingwie/Work/drag0n-php/tests/security/test/ok/symlink.txt) is not within the allowed path(s): (.) in /Users/Ingwie/Work/drag0n-php/tests/security/open_basedir.inc on line 47
007+ 
008+ Warning: rmdir(/Users/Ingwie/Work/drag0n-php/tests/security/test/ok): Directory not empty in /Users/Ingwie/Work/drag0n-php/tests/security/open_basedir.inc on line 55
009+ 
010+ Warning: rmdir(/Users/Ingwie/Work/drag0n-php/tests/security/test): Directory not empty in /Users/Ingwie/Work/drag0n-php/tests/security/open_basedir.inc on line 55
011+ 
012+ Warning: mkdir(): File exists in /Users/Ingwie/Work/drag0n-php/tests/security/open_basedir.inc on line 64
013+ bool(false)
014+ 
015+ Warning: mkdir(): File exists in /Users/Ingwie/Work/drag0n-php/tests/security/open_basedir.inc on line 65
016+ bool(false)
017+ bool(true)
018+ 
019+ Warning: symlink(): open_basedir restriction in effect. File(/Users/Ingwie/Downloads/php-5.5.3/tests/security/test/bad/bad.txt) is not within the allowed path(s): (.) in /Users/Ingwie/Work/drag0n-php/tests/security/open_basedir_linkinfo.php on line 12
020+ bool(false)
003- bool(true)
004- bool(true)
005- bool(true)
006- bool(true)
007- int(%d)
016- bool(true)
025+ int(16777218)
030+ 
031+ Warning: unlink(): open_basedir restriction in effect. File(/Users/Ingwie/Work/drag0n-php/tests/security/test/ok/symlink.txt) is not within the allowed path(s): (.) in /Users/Ingwie/Work/drag0n-php/tests/security/open_basedir_linkinfo.php on line 26
032+ bool(false)