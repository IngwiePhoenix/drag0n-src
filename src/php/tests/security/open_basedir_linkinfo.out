*** Testing open_basedir configuration [linkinfo] ***
bool(true)

Warning: is_dir(): open_basedir restriction in effect. File(/Users/Ingwie/Work/drag0n-php/tests/security/test/ok/symlink.txt) is not within the allowed path(s): (.) in /Users/Ingwie/Work/drag0n-php/tests/security/open_basedir.inc on line 43

Warning: unlink(): open_basedir restriction in effect. File(/Users/Ingwie/Work/drag0n-php/tests/security/test/ok/symlink.txt) is not within the allowed path(s): (.) in /Users/Ingwie/Work/drag0n-php/tests/security/open_basedir.inc on line 47

Warning: rmdir(/Users/Ingwie/Work/drag0n-php/tests/security/test/ok): Directory not empty in /Users/Ingwie/Work/drag0n-php/tests/security/open_basedir.inc on line 55

Warning: rmdir(/Users/Ingwie/Work/drag0n-php/tests/security/test): Directory not empty in /Users/Ingwie/Work/drag0n-php/tests/security/open_basedir.inc on line 55

Warning: mkdir(): File exists in /Users/Ingwie/Work/drag0n-php/tests/security/open_basedir.inc on line 64
bool(false)

Warning: mkdir(): File exists in /Users/Ingwie/Work/drag0n-php/tests/security/open_basedir.inc on line 65
bool(false)
bool(true)

Warning: symlink(): open_basedir restriction in effect. File(/Users/Ingwie/Downloads/php-5.5.3/tests/security/test/bad/bad.txt) is not within the allowed path(s): (.) in /Users/Ingwie/Work/drag0n-php/tests/security/open_basedir_linkinfo.php on line 12
bool(false)
int(16777218)
int(16777218)
int(16777218)
int(16777218)
int(16777218)

Warning: symlink(): open_basedir restriction in effect. File(/Users/Ingwie/Downloads/php-5.5.3/tests/security/test/bad/bad.txt) is not within the allowed path(s): (.) in /Users/Ingwie/Work/drag0n-php/tests/security/open_basedir_linkinfo.php on line 24
bool(false)
int(16777218)

Warning: unlink(): open_basedir restriction in effect. File(/Users/Ingwie/Work/drag0n-php/tests/security/test/ok/symlink.txt) is not within the allowed path(s): (.) in /Users/Ingwie/Work/drag0n-php/tests/security/open_basedir_linkinfo.php on line 26
bool(false)
*** Finished testing open_basedir configuration [linkinfo] ***